### PR TITLE
Removing artifacts with encoders, decoders and discriminators.

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

File: article/_posts/2019-11-28-disentanglement.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    authors: "Haofu Liao, Wei-An Lin, Jianbo Yuan, S. Kevin Zhou, and Jiebo Luo"
+    title:   "Artifact Disentanglement Network forUnsupervised Metal Artifact Reduction"
+    venue:   "MICCAI 2019"
+pdf: "https://arxiv.org/abs/1906.01806.pdf"
+---
+
+
+# Highlights
+* Unsupervised method for removing artifacts from medical images.  
+* Unsupervised method for transfering artifacts from one image to another one.
```
Comment (jhlegarreta) [inline article/_posts/2019-11-28-disentanglement.md]: transfering : transferring

File: article/_posts/2019-11-28-disentanglement.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+as well as real and fake artifacted images.
+
+![](/article/images/disentanglement/sc04.jpg)
+
+The third term is a simple L1 loss (used to train an auto-encoder)
+
+![](/article/images/disentanglement/sc03.jpg)
+
+The fourth term is a cycle loss which is a L1 distance function between the input artifact-free image $$y$$ and the recontructed aftifact-free image obtained after adding artifacts to $$y$$
```
Comment (jhlegarreta) [inline article/_posts/2019-11-28-disentanglement.md]: recontructed aftifact-free : reconstructed atifact-free

File: article/_posts/2019-11-28-disentanglement.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+layout: review
+title: "Artifact Disentanglement Network forUnsupervised Metal Artifact Reduction"
+tags: autoencoder deep-learning
```
Comment (jhlegarreta) [inline article/_posts/2019-11-28-disentanglement.md]: May be the tag `adversarial` is also relevant/worthwhile adding.

File: article/_posts/2019-11-28-disentanglement.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+---
+layout: review
+title: "Artifact Disentanglement Network forUnsupervised Metal Artifact Reduction"
```
Comment (lemairecarl) [inline article/_posts/2019-11-28-disentanglement.md]: Missing space: forUnsupervised

File: article/_posts/2019-11-28-disentanglement.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+![](/article/images/disentanglement/sc05.jpg)
+
+The last term is an L1 loss to force the reconstructed images to be not too different from the input images
+
+![](/article/images/disentanglement/sc06.jpg)
+
+## Results
+
+Results are quite impressive, in par with those obtained by supervised methods.
```
Comment (lemairecarl) [inline article/_posts/2019-11-28-disentanglement.md]: on par

File: article/_posts/2019-11-28-disentanglement.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+![](/article/images/disentanglement/sc08.jpg)
+
+![](/article/images/disentanglement/sc10.jpg)
+![](/article/images/disentanglement/sc11.jpg)
```
Comment (lemairecarl) [inline article/_posts/2019-11-28-disentanglement.md]: This last image is not included in the commit

**Comment by lemairecarl** [comment]
Please merge master into your branch; it will fix the "All checks have failed" problem
(except if there is really a problem in your PR)

**Pull request merged**